app/views/state_file/questions/ny201/edit.html.erb

```diff
@@ -37,6 +37,9 @@
     <%= f.cfa_date_select :date_electronic_withdrawal, "Electronic funds withdrawal Date" %>
     <%= f.cfa_input_field :amount_electronic_withdrawal, "Electronic funds amount" %>
 
+    <% unless params[:return_to_review].nil? %>
+      <%= hidden_field_tag "return_to_review", params[:return_to_review] %>
+    <% end %>
```

Comment:
This is totally good, but if you'd prefer avoiding a double negative you could do:
```erb
<% if params[:return_to_review].present? %>
      <%= hidden_field_tag "return_to_review", params[:return_to_review] %>
<% end %>
```

Reply:
haha duh, yeah that's better will change